api/lib/domain/models/TargetProfileAttachableForAdmin.js

```diff
@@ -0,0 +1,8 @@
+class TargetProfileAttachableForAdmin {
```

Comment:
Pour moi pas besoin de préciser "for admin", c'est un modèle bien spécifique à notre besoin métier qui est différent du modèle de target profile. Notre contexte est présicé par l'adjectif "attachable". Peu de risque à être utilisé dans d'autres contextes.

```suggestion
class AttachableTargetProfile {

Reply:
Done !